components/providers.tsx

```diff
@@ -0,0 +1,9 @@
+"use client";
+
```

Comment:
Providers need to go in `layout.tsx`, but it's a server component, so you need to split them out into a separate file/component that declares `use client`. This is more next annoyingness.